plugin/plugins/badger2ds/badger2ds.go

```diff
@@ -0,0 +1,127 @@
+package badger2ds
+
```

Comment:
If Badger isn't going to fix their x86 issues (especially the compilation error), then we should split this package into two. One with a build directive for `386` (or easier just calling the file badger2ds_386.go) that just returns errors and the other with a `// +build !386` directive.

That should mostly get things to work, although to get tests to pass we'll also need to add an exception to not run the badger2 sharness tests on 386.

Reply:
Badger has fixed the build failures, which allows out builds to work.  Config changes on our side allow our tests to pass. So, we should be ok, and we are completely avoiding the mmap problem now so we do not need to issue the warning about 32bit.  Badger should fix their 32bit tests, but we are avoiding the cause of those failures (mmap + 32bit).